app/src/main/java/de/danoeh/antennapod/activity/AudioplayerActivity.java

```diff
@@ -28,6 +28,9 @@ protected void onResume() {
         super.onResume();
         if (TextUtils.equals(getIntent().getAction(), Intent.ACTION_VIEW)) {
             Intent intent = getIntent();
+            if (intent.getData() == null) {
+                return;
```

Comment:
This should probably display an error message (Toast?)

Reply:
in this case, we would have made a programming error by not constructing the intent correctly.
there isn't much we could show the user here. I rather not do anything than crash.
also, I wanted to get rid of the inspection complaining ;)

Reply:
Could also be a programming error in another app if AP is used as player